### PR TITLE
Add use of the runtime/default seccomp profile.

### PR DESCRIPTION
**Pull request by Strigix**
I've modified the container and pod securitycontexts to let this application make use of the runtime/default seccomp profile. During testing this did not seem to interrupt any functionality. Please confirm this if possible.
This is done in light of:

https://github.com/giantswarm/roadmap/issues/259

Drop a message on slack if you've got questions.

### CODE REVIEW

File: helm/deletion-blocker-operator/templates/psp.yaml [hunk 1]
```diff
@@ -2,6 +2,8 @@ apiVersion: policy/v1beta1
 kind: PodSecurityPolicy
 metadata:
   name: {{ include "resource.psp.name" . }}
+  annotations:
+    seccomp.security.alpha.kubernetes.io/allowedProfileNames: 'runtime/default'
```
Comment (whites11) [inline helm/deletion-blocker-operator/templates/psp.yaml:6]: same

File: helm/deletion-blocker-operator/values.schema.json [hunk 2]
```diff
@@ -54,92 +59,31 @@
...
-                        "required": [
-                            "group",
-                            "kind",
-                            "resource",
-                            "version"
-                        ]
-                    },
-                    "query": {
-                        "type": "string"
+            "type": "array"
```
Comment (erkanerol) [inline helm/deletion-blocker-operator/values.schema.json:73]: Please adapt this file manually and ensure that you are not deleting anything.

File: helm/deletion-blocker-operator/templates/psp.yaml (hunk 1 above)
Comment (erkanerol) [reply helm/deletion-blocker-operator/templates/psp.yaml:6]: what does this mean?

**Review by erkanerol — APPROVED** [review]

**Comment by erkanerol** [comment]
@Strigix I fixed the schema problem. Feel free to merge.

**Pull request merged**